localisation/english/russia2_localisation_l_english.yml

```
@@ -14280,3 +14355,118 @@
+russia2_russia.721.a:0 "It's our divine right"
+russia2_russia.721.b:0 "It's a bad idea"
+russia2_russia.722.t:0 "[FROM.GetNameDef] Accepts"
+russia2_russia.722.d:0 ""
+russia2_russia.722.a:0 "May our crowns shine"
+russia2_russia.723.t:0 "[FROM.GetNameDef] Declines"
+russia2_russia.723.d:0 ""
+russia2_russia.723.a:0 "They'll drown in their pride"
+russia2_russia.678.a:0 "Lets Go"
+ 
+
+
```

YAMLlint (1.37.1)

[error] too many blank lines (2 > 0)

(empty-lines)